src/NuGet.Clients/NuGet.VisualStudio.Common/IDE/VsHierarchyUtility.cs

```diff
@@ -32,9 +28,11 @@ public static string GetProjectPath(IVsHierarchy project)
             return projectPath;
         }
 
-        public static async Task<bool> IsSupportedAsync(IVsHierarchy hierarchy, string projectTypeGuid)
+        public static bool IsSupported(IVsHierarchy hierarchy, string projectTypeGuid)
```

Comment:
This is unused in product code as far as I can tell.
Can we delete it?

Reply:
I'm about to use it, and swap it for the DTE.Project one.